index.js

```diff
@@ -98,87 +141,91 @@ function myMap(array, callback) {
 }
 
 // ***** Callbacks - Question 1 *****
-
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 1", "color: red")
+console.log("%cCallbacks - Question 1", "color: red")
 
-// console.log(myMap([1,2,3,4,5], triple)) 
-// // => [3,6,9,12,15]
+function triple(num) {
+  return num * 3
+}
+console.log(myMap([1, 2, 3, 4, 5], triple))
+// => [3,6,9,12,15]
 
-// console.log(myMap([2,4,6,8], triple))   
-// // => [6,12,18,24]
-// console.log("%c----------", "color: red") 
+console.log(myMap([2, 4, 6, 8], triple))
+// => [6,12,18,24]
+console.log("%c----------", "color: red")
 
 
 // ***** Callbacks - Question 2 *****
 // *** Uncomment the lines below to test
-// console.log("%cCallbacks - Question 2", "color: red")
-
-// console.log(myMap(["Raffy", "Chase"], greet)) 
-// // => ["Hi, Raffy!", "Hi, Chase!"]
-// console.log("%c----------", "color: red") 
+console.log("%cCallbacks - Question 2", "color: red")
+function greet(name) {
+  return `Hi, ${name}!`
+}
+console.log(myMap(["Raffy", "Chase"], greet))
+// => ["Hi, Raffy!", "Hi, Chase!"]
+console.log("%c----------", "color: red")
 
 
 
 // ***** Scope & Closures *****
-
 function takeANumber(line, name) {
   line.push(name)
 
   return `Welcome, ${name}. You are number ${line.length} in line.`
 }
-
 function nowServing(line) {
   if (!line.length) {
     return "There is nobody waiting to be served!"
   }
   return `Currently serving ${line.shift()}.`
 }
-
 const line = []
 
 // ***** Scope & Closures - Question 1 *****
-
-
 // *** Uncomment the lines below to test
-// console.log("%cScope & Closures - Question 1", "color: red")
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 1`
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 2`
-
-// console.log(nowServing(line))
-// // => `Currently serving 1.`
-
-// console.log(nowServing(line))
-// // => `Currently serving 2.`
-
-// console.log(takeATicketNumber(line))
-// // => `Welcome. You are ticket number 3`
-
-// console.log("%c----------", "color: red") 
+console.log("%cScope & Closures - Question 1", "color: red")
+function ticketNumberMaker(line) {
+  nextNum = 1
+  return function ticketMaker(line) {
+    line.push(nextNum)
+    nextNum++
+    return `Welcome. You are ticket number ${nextNum - 1}`
+  }
+}
```

Comment:
Looks like this the solution for Question 2; looks good! One thing you could do to simplify the logic is initialize `nextNum` to 0 instead of 1 on line 188. Like so:
```suggestion
function ticketNumberMaker(line) {
  nextNum = 0
  return function ticketMaker(line) {
    nextNum++
    line.push(nextNum)
    return `Welcome. You are ticket number ${nextNum}`
  }
}
```